README.md

```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #12 - Tip to measure performance of a javascript block
+
+2016-01-12 by [@manmadareddy](https://twitter.com/manmadareddy)
+
+For quickly measuring performance of a javascript block, we can use the console functions like
+console.time(label) and console.timeEnd(label).
+
+```javascript
+console.time("Array initialize");
+var arr = new Array(100),
+    len = arr.length,
+    i;
+
+for (i = 0; i < len; i++) {
+    arr[i] = new Object();
+};
+console.timeEnd("Array initialize"); // Outputs: Array initialize: 0.711ms
+```
+
+More info about console and javascript-benchmarking:
+
+https://github.com/DeveloperToolsWG/console-object
+https://mathiasbynens.be/notes/javascript-benchmarking
+
+Demo: (outputs in browser console)
```

Comment:
Update by

Demo: [jsfiddle](https://jsfiddle.net/meottb62/
) - [codepen](http://codepen.io/anon/pen/JGJPoa) (outputs in browser console)